package/utils/wtmcu/Makefile

```diff
@@ -0,0 +1,46 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=wtmcu
+PKG_VERSION:=0.1.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://gitlab.com/evgenyz/$(PKG_NAME)/-/archive/v$(PKG_VERSION)
```

Comment:
This doesn't seem to be much code. I'd prefer to have this code in OpenWrt repo instead of having to rely on a private external source with only 2 commits and < 10 files.

Reply:
Come on, it is GitLab, the largest public GIT repository for open-source projects not owned by Mirosoft. It is hardly private. 90% of OpenWrt packages are like that. I can move it somewhere else (even into the OpenWrt repo) if you like.

It is inactive because it is not used by anything (since this PR is not merged). I expect a bit of action there once it would be used and tested wider. Also, it is not feature-complete in regard of MCU capabilities. Not sure if it would be convenient to work on the daemon if it would be embedded in OpenWrt, but I would rely on your opinion here.

Reply:
Yes, and if you delete it, it's gone and we have a "broken" device in our tree.
"private" was mostly referring to that it's under your personal control only.

Anyway, again this indicates the disadvantages of relying onto this. Obviously, if this package was moved into OpenWrt, we would have the maintenance burden, and most probably nobody would maintain it if you were occupied otherwise or just wouldn't care anymore in two years (without any offense).

So, as discussed in this PR initially, including this packages really is a major drawback for this proposal, no matter where we put it.

Reply:
Well, this things could happen, we all mortal in any case.

So I tried my best to make it as harmless as possible. It is BSD-licensed and public. It has almost no dependencies. Chances that it will become broken because of FW update for MCU chip are nearly zero. I also tried to make it as simple as possible while having all the knobs, including ubus support; it'd be easy to take over for the next person interested in this device.

I can transfer it anywhere and make necessary legal modifications for it to be as much independent from me as possible, I have no intention to have a dictatorship over it.

And, of course, right now this device is as broken as it could possibly be because is is not supported at all :)

Reply:
> And, of course, right now this device is as broken as it could possibly be because is is not supported at all :)

Yes, and as long as we do not support it, we don't have to care :-)

Reply:
> > And, of course, right now this device is as broken as it could possibly be because is is not supported at all :)
> 
> Yes, and as long as we do not support it, we don't have to care :-)

This is also a legitimate way of thinking. And I fully understand it, despite the fact that it just so happened I have a different one in this particular case :)